packages/app-check-types/index.d.ts

```diff
@@ -28,17 +28,35 @@ export interface FirebaseAppCheck {
     siteKeyOrProvider: string | AppCheckProvider,
     isTokenAutoRefreshEnabled?: boolean
   ): void;
+
+  /**
+   *
+   * @param isTokenAutoRefreshEnabled - If set to true, enables SDK to
+   * automatically refresh AppCheck token as needed. If false, disables this
+   * functionality. This overrides any value set during `activate()`.
+   */
   setTokenAutoRefreshEnabled(isTokenAutoRefreshEnabled: boolean): void;
 }
 
+/**
+ * An AppCheck provider. This may be either the built-in reCAPTCHA provider
+ * or a custom provider. For more on custom providers, see
+ * https://firebase.google.com/docs/app-check/web-custom-provider
```

Comment:
Looks like this will 404 until 5/13:

https://critique-ng.corp.google.com/cl/371400287

Reply:
We can hold off on publishing our reference docs until 5/13 also, then. You can make a CL dependent on another CL, right?

Reply:
Makes sense, thanks!